README.md

```diff
@@ -1 +1,109 @@
-# angular-concepts-readme
+# Angular Core Concepts
```

Comment:
Based on the content here, 'MVVM and Data Binding in Angular' would be a better title